99e323b84e4d36b304a9275c29f737c53f19c543: Pulling from gr0v1ty/nano-node
6414378b6477: Pulling fs layer
83683acd158d: Pulling fs layer
b16dc5b00cfb: Pulling fs layer
f48a5f9c25a9: Pulling fs layer
35c0517173b1: Pulling fs layer
75e56b031a7d: Pulling fs layer
a6157e48a493: Pulling fs layer
a601886b28c4: Pulling fs layer
4f4fb700ef54: Pulling fs layer
7705f124a6dd: Pulling fs layer
e2671878dc12: Pulling fs layer
75e56b031a7d: Waiting
a6157e48a493: Waiting
a601886b28c4: Waiting
4f4fb700ef54: Waiting
7705f124a6dd: Waiting
e2671878dc12: Waiting
f48a5f9c25a9: Waiting
35c0517173b1: Waiting
83683acd158d: Verifying Checksum
83683acd158d: Download complete
6414378b6477: Verifying Checksum
6414378b6477: Download complete
f48a5f9c25a9: Verifying Checksum
f48a5f9c25a9: Download complete
35c0517173b1: Verifying Checksum
35c0517173b1: Download complete
75e56b031a7d: Download complete
a6157e48a493: Verifying Checksum
a6157e48a493: Download complete
a601886b28c4: Verifying Checksum
a601886b28c4: Download complete
4f4fb700ef54: Verifying Checksum
4f4fb700ef54: Download complete
e2671878dc12: Download complete
7705f124a6dd: Verifying Checksum
7705f124a6dd: Download complete
6414378b6477: Pull complete
83683acd158d: Pull complete
b16dc5b00cfb: Verifying Checksum
b16dc5b00cfb: Download complete
b16dc5b00cfb: Pull complete
f48a5f9c25a9: Pull complete
35c0517173b1: Pull complete
75e56b031a7d: Pull complete
a6157e48a493: Pull complete
a601886b28c4: Pull complete
4f4fb700ef54: Pull complete
7705f124a6dd: Pull complete
e2671878dc12: Pull complete
Digest: sha256:5d0da6136ab1915800225e57828fd6c59bb8421aa612af63fbb2b4cd08fb1761
Status: Downloaded newer image for gr0v1ty/nano-node:99e323b84e4d36b304a9275c29f737c53f19c543
docker.io/gr0v1ty/nano-node:99e323b84e4d36b304a9275c29f737c53f19c543
https://api.github.com/repos/gr0vity-dev/nanolab-configs/contents/quick_testing/5n4pr_conf_change_dependant.json
Start Fetching '5n4pr_conf_change_dependant_config.json' to '/home/runner/work/nano-node-builder/nano-node-builder/testcases/5n4pr_conf_change_dependant_config.json'
Done  Fetching '5n4pr_conf_change_dependant_config.json' to '/home/runner/work/nano-node-builder/nano-node-builder/testcases/5n4pr_conf_change_dependant_config.json'
Start Fetching '5n4pr_bucket0-1-88-90-100_10kaccs.ldb' to '/home/runner/work/nano-node-builder/nano-node-builder/testcases/downloads/5n4pr_bucket0-1-88-90-100_10kaccs.ldb'
Done  Fetching '5n4pr_bucket0-1-88-90-100_10kaccs.ldb' to '/home/runner/work/nano-node-builder/nano-node-builder/testcases/downloads/5n4pr_bucket0-1-88-90-100_10kaccs.ldb'
Start Fetching '5n4pr_quicktesting_config.toml' to '/home/runner/work/nano-node-builder/nano-node-builder/testcases/downloads/5n4pr_quicktesting_config.toml'
Done  Fetching '5n4pr_quicktesting_config.toml' to '/home/runner/work/nano-node-builder/nano-node-builder/testcases/downloads/5n4pr_quicktesting_config.toml'
Error executing command: nanomock create && nanomock reset
Error: INFO: nanomock data has been copied to your current working directory.
INFO: Log level : default
INFO: Log level : default
INFO: Log level : default
INFO: Log level : default
INFO: Log level : default
SUCCESS: Docker Compose file created at /home/runner/work/nano-node-builder/nano-node-builder/nano_nodes/docker-compose.yml
SUCCESS: None
SUCCESS: init_containers completed with None result.
SUCCESS: stop_containers completed with None result.
SUCCESS: reset_nodes_data completed with None result.
The network 'nl_nano-local' already exists.
Error executing command: nanomock create
Error: INFO: nanomock data has been copied to your current working directory.
INFO: Log level : default
INFO: Log level : default
INFO: Log level : default
INFO: Log level : default
INFO: Log level : default
SUCCESS: Docker Compose file created at /home/runner/work/nano-node-builder/nano-node-builder/nano_nodes/docker-compose.yml
SUCCESS: None
SUCCESS: init_containers completed with None result.
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
Error executing command: nanomock start && nanomock init_wallets
Error: WARNING: ['nl_genesis', 'nl_pr1', 'nl_pr2', 'nl_pr3', 'nl_pr4']
INFO: Nodes ['nl_genesis', 'nl_pr1', 'nl_pr2', 'nl_pr3', 'nl_pr4'] reachable
SUCCESS: start_containers completed with None result.
WARNING: ['nl_genesis']
WARNING: ['nl_pr1']
WARNING: ['nl_pr2']
WARNING: ['nl_pr3']
WARNING: ['nl_pr4']
INFO: Nodes ['nl_genesis'] reachable
INFO: Nodes ['nl_pr3'] reachable
INFO: Nodes ['nl_pr2'] reachable
INFO: Nodes ['nl_pr4'] reachable
INFO: Nodes ['nl_pr1'] reachable
INFO: WALLET 6BABDE50D3DC7ED22A63ADA7941CBBAF140AC1F26B2EDC6A0F9276966ABB625A CREATED FOR nl_pr2 WITH ACCOUNT nano_3sz3bi6mpeg5jipr1up3hotxde6gxum8jotr55rzbu9run8e3wxjq1rod9a6
INFO: WALLET 2AD85E083D641297935DB69C7EFE0D1D59422374CE481B55D3408396CA674749 CREATED FOR nl_pr3 WITH ACCOUNT nano_3z93fykzixk7uoswh8fmx7ezefdo7d78xy8sykarpf7mtqi1w4tpg7ejn18h
INFO: WALLET 8246026390BC135C5BFA62E7266BACBD38C4D4E8119C1A5CF298C5F8B5430820 CREATED FOR nl_genesis WITH ACCOUNT nano_1fzwxb8tkmrp8o66xz7tcx65rm57bxdmpitw39ecomiwpjh89zxj33juzt6p
INFO: WALLET F427300C521292DF3B2495D89FC21459D7908C7AF17F0E4D2B14DF7AE657A837 CREATED FOR nl_pr1 WITH ACCOUNT nano_1ge7edbt774uw7z8exomwiu19rd14io1nocyin5jwpiit3133p9eaaxn74ub
INFO: WALLET 3235707CB16F41606E06E2DB81A6D2B3D334B0A52FC19856F2A45D3042B77FB1 CREATED FOR nl_pr4 WITH ACCOUNT nano_137xfpc4ynmzj3rsf3nej6mzz33n3f7boj6jqsnxpgqw88oh8utqcq7nska8
SUCCESS: ['WALLET 6BABDE50D3DC7ED22A63ADA7941CBBAF140AC1F26B2EDC6A0F9276966ABB625A CREATED FOR nl_pr2 WITH ACCOUNT nano_3sz3bi6mpeg5jipr1up3hotxde6gxum8jotr55rzbu9run8e3wxjq1rod9a6', 'WALLET 2AD85E083D641297935DB69C7EFE0D1D59422374CE481B55D3408396CA674749 CREATED FOR nl_pr3 WITH ACCOUNT nano_3z93fykzixk7uoswh8fmx7ezefdo7d78xy8sykarpf7mtqi1w4tpg7ejn18h', 'WALLET 8246026390BC135C5BFA62E7266BACBD38C4D4E8119C1A5CF298C5F8B5430820 CREATED FOR nl_genesis WITH ACCOUNT nano_1fzwxb8tkmrp8o66xz7tcx65rm57bxdmpitw39ecomiwpjh89zxj33juzt6p', 'WALLET F427300C521292DF3B2495D89FC21459D7908C7AF17F0E4D2B14DF7AE657A837 CREATED FOR nl_pr1 WITH ACCOUNT nano_1ge7edbt774uw7z8exomwiu19rd14io1nocyin5jwpiit3133p9eaaxn74ub', 'WALLET 3235707CB16F41606E06E2DB81A6D2B3D334B0A52FC19856F2A45D3042B77FB1 CREATED FOR nl_pr4 WITH ACCOUNT nano_137xfpc4ynmzj3rsf3nej6mzz33n3f7boj6jqsnxpgqw88oh8utqcq7nska8']
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
nanorpc.versions.v25_1
nanorpc.versions.v26_0
nanorpc.versions.v26_1
.DEBUG 1 unconfirmed blocks! e.g. 925ED16481B1F0FED5C3BF8ABCD2EE28150117557A0389E8062D1170A0360A61
.DEBUG 1 unconfirmed blocks! e.g. 1CADB41C73F3EE1D0F62861CE3BC10F4DB66838BF030B5106411ABA927988CA0
.Exception in thread Thread-1:
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/command/mixins/threaded_command_mixin.py", line 26, in _execute_command_sequence
    self.command_instance.execute_another_command(command_config)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/command/command.py", line 49, in execute_another_command
    another_command.execute()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/command/command.py", line 41, in execute
    self.mixin.execute()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/command/mixins/python_command_mixin.py", line 40, in execute
    method(**filtered_variables)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/pycmd.py", line 35, in publish_create
    asyncio.run(test_case.run())
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/publisher/test_case.py", line 73, in run
    await self._generate_and_confirm_block(seed_index)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/publisher/test_case.py", line 59, in _generate_and_confirm_block
    res = await self.bg_l.blockgen_single_change(source_seed=self.source_seed,
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanolab/publisher/block_generator.py", line 56, in blockgen_single_change
    return await self.nano_rpc_default.create_change_block(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanomock/modules/nl_rpc.py", line 172, in create_change_block
    return await self.get_block_result(block,
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanomock/modules/nl_rpc.py", line 354, in get_block_result
    publish = await self._try_publish_block(block, start_time, exit_after_s)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanomock/modules/nl_rpc.py", line 388, in _try_publish_block
    publish = await self.process(block["block"], json_block=True)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanomock/modules/nl_rpc.py", line 58, in process
    return await self.nanorpc.process(block, force=force, subtype=subtype, json_block=json_block, async_=async_)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanorpc/client.py", line 200, in process
    return await self.rpc.process(block, force=force, subtype=subtype, json_block=json_block, async_=async_)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanorpc/client_dynamic.py", line 33, in method
    return await self.process_payloads([payload])
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanorpc/client_dynamic.py", line 119, in process_payloads
    return await self._request_with_retry(self._request, payloads)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanorpc/client_dynamic.py", line 101, in _request_with_retry
    return await self._retry_on_exception(coroutine,
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nanorpc/client_dynamic.py", line 95, in _retry_on_exception
    raise MaxRetriesExceededError(
nanorpc.client_dynamic.MaxRetriesExceededError: All 1 retries exhausted for _request. Last error: 
